docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -0,0 +1,342 @@
+  fetchInViewOptions,
+  routeInViewOptions,
+}) => {
+  const Wrapper = ({ children }) => {
+    const { state } = useConnect();
+
+    const current = state.source.get(link);
+    const next =
+      isArchive(current) && current.next
+        ? state.source.get(current.next)
+        : null;
+
+    const { supported, fetchRef, routeRef } = useInfiniteScroll({
+      currentLink: link,
+      nextLink: next?.link,
+      fetchInViewOptions,
+      routeInViewOptions,
+    });
+
+    if (!current.isReady || isError(current)) return null;
+    if (!supported) return children;
+
+    const container = css`
+      position: relative;
+    `;
+
+    const fetcher = css`
+      position: absolute;
+      width: 100%;
+      bottom: 0;
+    `;
+
+    return (
+      <div css={container} ref={routeRef}>
+        {children}
+        {<div css={fetcher} ref={fetchRef} />}
+      </div>
+    );
+  };
+
+  return connect(Wrapper);
+};
+```
+
+## `useArchiveInfiniteScroll`
+
+This hook implements the logic needed to include infinite scroll in archives (i.e. categories, tags, the posts archive, etc.).
+
+The hook receives options to set a limit of pages shown automatically, to disable it, and also settings for the intersection observers that are passed to the `useInfiniteScroll` hooks used internally.
+
+`useArchiveInfiniteScroll` is designed to be used inside an `Archive` component. That component would render all the archive pages from the `pages` returned by the hook.
+
+In addition to the above, the hook returns a set of boolean values that indicate if the next page is being fetched, if the limit has been reached, or if the next page returned an error, and a function that allows the next page to be fetched manually.
+
+### Parameters
+
+It accepts an optional object with the following props:
+
+| Name                     | Type                | Default    | Required | Description                                                                                                                                                                                                              |
+| :----------------------- | :------------------ | :--------- | :------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`active`**             | boolean             | `true`     | no       | A boolean indicating if this hook should be active or not. It can be useful in situations where users want to share the same component for different types of Archives, but avoid doing infinite scroll in some of them. |
+| **`limit`**              | number              | `infinite` | no       | The number of pages that the hook should load automatically before switching to manual fetching.                                                                                                                         |
```

Comment:
The default value here should be [`Infinity`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity) which is the global property representing infinity.